core/src/Modal.html

```diff
@@ -0,0 +1,134 @@
+<div class="fd-ui__overlay fd-overlay fd-overlay--modal" style="z-index:999">
+  <div class="fd-modal">
+    <div class="fd-modal__content" style="height:100%">
+      <div class="fd-modal__header">
+        <h1 class="fd-modal__title">{modalSettings.title}</h1>
+        <button
+          class="fd-button--light fd-modal__close"
+          on:click="fire('close')"
+          aria-label="close"
+        ></button>
+      </div>
+      <div class="iframeModalCtn"></div>
+    </div>
+  </div>
+</div>
+<script>
+  import * as Navigation from './navigation/services/navigation';
+  import * as RoutingHelpers from './utilities/helpers/routing-helpers';
+  import { LuigiConfig } from './services/config';
+  import * as GenericHelpers from './utilities/helpers/generic-helpers';
+
+  const prepareNodeData = async (component, path) => {
+    const pathUrlRaw =
+      path && path.length ? GenericHelpers.getPathWithoutHash(path) : '';
+    const pathData = await Navigation.getNavigationPath(
+      LuigiConfig.getConfigValueAsync('navigation.nodes'),
+      path
+    );
+    const params = RoutingHelpers.parseParams(pathUrlRaw.split('?')[1]);
+    const nodeParams = RoutingHelpers.getNodeParams(params);
+    const lastNode = RoutingHelpers.getLastNodeObject(pathData);
+    const modalSettings = component.get().modalSettings;
+    if (!modalSettings.title) {
+      modalSettings.title = lastNode.label;
+    }
+    component.get().isDataPrepared = true;
+    component.set({ modalSettings, lastNode, pathData, nodeParams });
+  };
+
+  const getNode = async (component, path) => {
+    if (component.get().isDataPrepared) {
+      const { nodeParams, lastNode, modalSettings, pathData } = component.get();
+      const iframe = createIframeModal(
+        lastNode.viewUrl,
+        {
+          context: pathData.context,
+          pathParams: pathData.pathParams,
+          nodeParams
+        },
+        component
+      );
+      component.root.set({
+        modalIframe: iframe,
+        modalIframeData: { ...pathData, nodeParams }
+      });
+    } else {
+      await prepareNodeData(component, path);
+    }
+  };
+  const createIframeModal = (viewUrl, componentData, component) => {
+    const elem = document.getElementsByClassName('fd-modal');
+    let sizeInPercent = '80%';
```

Comment:
The default value is cool 👌 but please mention it in Luigi API (docu)

Reply:
Also, I'd change the variable name or its values. You double the "percent" thing.

Reply:
👍